inginious/frontend/templates/course_user_settings.html

```diff
@@ -0,0 +1,69 @@
+{% set staff = user_manager.has_staff_rights_on_course(course) %}
+{% block column %}
+    {{ template_helper.call('course_menu', course=course, template_helper=template_helper) | safe }}
+    {%if staff %}
+        {% if not course.is_open_to_non_staff() %}
+            <div class="alert alert-warning" role="alert">
+                {{ _("This course is currently invisible for students. You can change this by modifying the \"accessibility\" option in the configuration of the course.") }}
+            </div>
+        {% endif %}
+        <div class="list-group">
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/admin/{{ course.get_id() }}">
+                <i class="fa fa-user-secret fa-fw"></i>&nbsp; {{ _("Course administration") }}
+            </a>
+        </div>
+    {% elif registered %}
+        <div class="list-group">
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/group/{{ course.get_id() }}">
+                <i class="fa fa-group fa-fw"></i>&nbsp;
+                {% set mygroup = user_manager.get_course_user_group(course) %}
+                {% if mygroup and user_manager.session_username() in mygroup['students'] %}
+                    {{ _("Group : {}").format(mygroup['description']) }}
+                {% else %}
+                    {{ _("Group management") }}
+                {% endif %}
+            </a>
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/user_settings/{{ course.get_id() }}">
+                <i class="fa fa-cogs fa-fw"></i>&nbsp; {{ _("Course Settings") }}
+            </a>
+        </div>
+    {%endif %}
+{% endblock %}
+{% block navbar %}
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            {% if registered %}
+                <li class="breadcrumb-item"><a href="{{ get_homepath() }}/mycourses" title="{{ _('My courses') }}" data-toggle="tooltip" data-placement="bottom"><i class="fa fa-th-list"></i></a></li>
+            {% else %}
+                <li class="breadcrumb-item"><a href="{{get_homepath() }}/courselist" title="{{ _('Course list') }}" data-toggle="tooltip" data-placement="bottom"><i class="fa fa-th-list"></i></a></li>
+            {% endif %}
+            <li class="breadcrumb-item"><a href="{{get_homepath() }}/course/{{ course.get_id() }}">{{ course.get_name(user_manager.session_language()) }} <span class="sr-only">{{ _("(current)") }}</span></a></li>
+            <li class="breadcrumb-item active"><a href="#"><i class="fa fa-cogs"></i> {{ _("Course settings") }}</a></li>
+        </ol>
+    </nav>
+{% endblock %}
+{% block content %}
+    {%  if feedback is not none %}
+        <div class="alert alert-{{ feedback[0] }}" role="alert">
+            {{ _(feedback[1] ) }}
+        </div>
+    {% endif %}
+    <h2>{{_("Course settings")}}</h2>
+    <form method="post" action="">
```

Comment:
An ``alert-info`` block telling the user *there is no customizable option for this course* in case no settings in defined would be great. It would also be simpler than hiding the page and restrict access in this case.